opentelemetry-sdk/src/logs/config.rs

```diff
@@ -1,5 +1,10 @@
 use std::borrow::Cow;
 
+/// Default trace configuration
+pub fn config() -> Config {
```

Comment:
Why? Using `Config::default()` seems like an idiomatic convention, this is more opaque.

Reply:
Because this is what the `tracing` module provides: https://github.com/open-telemetry/opentelemetry-rust/blob/0a893ce7dceb182ba443a944d49ac7804237bbe9/opentelemetry-sdk/src/trace/config.rs#LL12C1-L15C2

My goal is to bring logs and tracing a little bit closer so I don't have to write as much duplicate code.

Reply:
Okay, how about we remove this on the `tracing` side?

Reply:
I'm not going to make such a breaking change in this PR. My goal is only to bring logs up to where tracing is, before logs gets widely depended on and changes become harder to make.